provider/pkg/gen/typegen.go

```diff
@@ -693,6 +705,14 @@ func createGroups(definitionsJSON map[string]any, allowHyphens bool) []GroupConf
 
 				return fmt.Sprintf("%s/%s", gStripped, v)
 			}
+
+			// These resources are hard-coded as lists as they do not adhere to the normal conventions.
```

Comment:
The list is nested under the `resources` and `groups` field instead of `items` for these two kinds.